qa/distros/all/ubuntu_18.04_podman.yaml

```diff
@@ -1,11 +1,12 @@
 os_type: ubuntu
 os_version: "18.04"
 
-# feel free to remove this test, if ppa:projectatomic is no longer maintained.
+# Accoring to https://podman.io/getting-started/installation.html
 tasks:
 - exec:
     all:
-    - sudo apt -y install software-properties-common
-    - sudo add-apt-repository -y ppa:projectatomic/ppa
+    - sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
+    - wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_18.04/Release.key -O- | sudo apt-key add -
```

Comment:
2020-02-17 09:21:44 ERROR 404: Not Found.

Reply:
might be a problem with yaml escaping.

Reply:
```yaml
os_type: ubuntu
os_version: '18.04'
tasks:
- exec:
    all: ['sudo sh -c "echo ''deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04/
        /'' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list" ', 'wget
        -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_18.04/Release.key
        -O- | sudo apt-key add -', sudo apt-get update -qq, sudo apt -y install podman,
      'echo -e "[registries.search]

        registries = [''docker.io'']" | sudo tee /etc/containers/registries.conf']

```
very ugly indeed.